faucet/valve.py

```diff
@@ -1021,14 +1045,13 @@ def _get_config_changes(self, new_dp):
         for vid, new_vlan in list(new_dp.vlans.items()):
             if vid not in self.dp.vlans or new_vlan != self.dp.vlans[vid]:
                 changed_vlans.add(vid)
-            for port in new_vlan.get_ports():
-                changed_ports.add(port.number)
+                for port in new_vlan.get_ports():
+                    changed_ports.add(port.number)
```

Comment:
This was a bug. Without the whitespace, we reload all ports no matter what their change status